game/scripts/vscripts/modifiers/player/modifier_player_76.lua

```diff
@@ -6,7 +6,7 @@ function modifier_player_76:GetTexture() return "player/76" end
 
 function modifier_player_76:OnCreated()
 	self.iMoveSpeed = 176
-	self.iManaRegen = 1.76
+	self.Bonus_Mana_Regen_Total_Percentage = 1.76
```

Comment:
这里不需要变吧，变不变应该不影响实际运行。估计把变量名风格统一了，改成`iManaRegenPercent`可能好点？
```suggestion
	self.iManaRegenPercent= 1.76
```